packages/data-stores/src/site/actions.ts

```diff
@@ -294,13 +294,22 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		} );
 	}
 
-	function* setDesignOnSite( siteSlug: string, selectedDesign: Design, siteVerticalId?: string ) {
+	function* setDesignOnSite(
+		siteSlug: string,
+		selectedDesign: Design,
+		selectedStyleVariation?: StyleVariation,
+		siteVerticalId?: string
```

Comment:
Sorry for that late party. But it might be better to make the non-required parameters be an object, then we don't need to do something like this

```js
setDesignOnSite( siteSlugOrId, _selectedDesign, undefined, siteVerticalId )
```

But it's not a blocker for this PR.

Reply:
I think this is a really good suggestion, thanks! I decided to implement it in this PR, could you recheck?